pkgs/tools/inputmethods/interception-tools/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl, fetchFromGitHub, pkgconfig, cmake, libyamlcpp, libevdev,
+  libudev }:
+
+let
+  libyamlcppWithoutBoost = libyamlcpp.overrideAttrs (oldAttrs: rec {
```

Comment:
This means, it also needs `libyaml` without boost and not just a newer version?

Reply:
I'm assuming you meant `libyamlcpp` because `libyaml` is for C, so it can't use boost anyway since boost is a c++ library.

But yeah so `libyamlcpp` is just a dependency for `interception-tools`, and `boost` is a dependency for the latest release of `libyamlcpp`. So `interception-tools` doesn't directly need boost at all, if that makes sense. However, the boost problem obviously disappears if I use a more recent *git* version of libyamlcpp instead, because they got rid of the boost dependency since their last release. So I thought that I should just do that. (And also, the latest release of libyamlcpp was in January 2016 too, so it's been a while.)

I guess, are you saying that `libyamlcppWithoutBoost` is a misleading name? In that case, maybe `libyamlcppGit` then? Or any suggestions? I don't know if there is a convention here.

Reply:
I thought, there is some kind of optional boost support in the library.

Reply:
I would move this expression next to `libyamlcpp` in `all-packages.nix`, with a comment that it is required by `interception-tools`. This way, we can easier remove it again, when there is a `libyamlcpp` update.

Reply:
Ok, done. :+1: